scalatest/src/main/scala/org/scalatest/tools/ScalaTestFramework.scala

```diff
@@ -499,7 +502,7 @@ Tags to include and exclude: -n "CheckinTests FunctionalTests" -l "SlowTests Net
             def testName = tn
             def description = tn
             def result = r
-            def error = e getOrElse null
+            def error = (e getOrElse null).asInstanceOf[Throwable]
```

Comment:
**Overridden java method return type cannot be null**
The nullness in the return type of all java methods are cast away by the `-Yjava-interop-dont-nullify-outermost` compiler flag. `asInstanceOf[A]` is used for return values of type `A | Null`.